examples/python/src/authors/query.py

```diff
@@ -1,92 +1,77 @@
 # Code generated by sqlc. DO NOT EDIT.
-from typing import AsyncIterator, Awaitable, Iterator, Optional, overload
+from typing import AsyncIterator, Iterator, Optional
 
-import sqlc_runtime as sqlc
+import sqlalchemy
+import sqlalchemy.ext.asyncio
 
 from authors import models
 
 
-CREATE_AUTHOR = """-- name: create_author :one
+CREATE_AUTHOR = """-- name: create_author \\:one
 INSERT INTO authors (
           name, bio
 ) VALUES (
-  $1, $2
+  :p1, :p2
 )
 RETURNING id, name, bio
 """
 
 
-DELETE_AUTHOR = """-- name: delete_author :exec
+DELETE_AUTHOR = """-- name: delete_author \\:exec
 DELETE FROM authors
-WHERE id = $1
+WHERE id = :p1
 """
 
 
-GET_AUTHOR = """-- name: get_author :one
+GET_AUTHOR = """-- name: get_author \\:one
 SELECT id, name, bio FROM authors
-WHERE id = $1 LIMIT 1
+WHERE id = :p1 LIMIT 1
 """
 
 
-LIST_AUTHORS = """-- name: list_authors :many
+LIST_AUTHORS = """-- name: list_authors \\:many
 SELECT id, name, bio FROM authors
 ORDER BY name
 """
 
 
-@overload
-def create_author(conn: sqlc.Connection, name: str, bio: Optional[str]) -> Optional[models.Author]:
-    pass
+class Query:
```

Comment:
I used the name of the sql file here, but I'm not sure that's the best option. Using `Querier` as a static name seems fine, although it means all the query files will have the same class names. That's not necessarily an issue as long as the module itself is imported instead of just the query class (so `from package import query` and `query.Querier()` instead of `from package.query import Querier` and `Querier()`).